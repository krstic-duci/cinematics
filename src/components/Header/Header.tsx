import Navigation from "components/Navigation";

import tmdb from "assets/images/tmdb.svg";

import styles from "./Header.module.css";

const Header = () => {
  return (
    <>
      <header>
        <Navigation />

        <div className="d-flex align-items-center justify-content-between">
          <h1 className={`${styles.h1} mb-0`}>Cinematics</h1>
          <img src={tmdb} alt="tmdb" width="100px" height="100px" />
        </div>
      </header>
    </>
  );
};

export default Header;
