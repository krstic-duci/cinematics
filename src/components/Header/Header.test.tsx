import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";

import Header from "./Header";

describe("Header component", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    expect(tree).toMatchSnapshot();
  });
});
