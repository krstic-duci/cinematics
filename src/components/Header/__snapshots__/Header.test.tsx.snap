// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<header>
  <div
    className="mt-2 align-items-center nav"
    onKeyDown={[Function]}
  >
    <div
      className="navItem nav-item"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <svg
          fill="currentColor"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z"
            fillRule="evenodd"
          />
          <path
            d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z"
            fillRule="evenodd"
          />
        </svg>
        Home
      </a>
    </div>
    <div
      className="navItem nav-item"
    >
      <a
        aria-current={null}
        href="/watch-later"
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8.515 1.019A7 7 0 0 0 8 1V0a8 8 0 0 1 .589.022l-.074.997zm2.004.45a7.003 7.003 0 0 0-.985-.299l.219-.976c.383.086.76.2 1.126.342l-.36.933zm1.37.71a7.01 7.01 0 0 0-.439-.27l.493-.87a8.025 8.025 0 0 1 .979.654l-.615.789a6.996 6.996 0 0 0-.418-.302zm1.834 1.79a6.99 6.99 0 0 0-.653-.796l.724-.69c.27.285.52.59.747.91l-.818.576zm.744 1.352a7.08 7.08 0 0 0-.214-.468l.893-.45a7.976 7.976 0 0 1 .45 1.088l-.95.313a7.023 7.023 0 0 0-.179-.483zm.53 2.507a6.991 6.991 0 0 0-.1-1.025l.985-.17c.067.386.106.778.116 1.17l-1 .025zm-.131 1.538c.033-.17.06-.339.081-.51l.993.123a7.957 7.957 0 0 1-.23 1.155l-.964-.267c.046-.165.086-.332.12-.501zm-.952 2.379c.184-.29.346-.594.486-.908l.914.405c-.16.36-.345.706-.555 1.038l-.845-.535zm-.964 1.205c.122-.122.239-.248.35-.378l.758.653a8.073 8.073 0 0 1-.401.432l-.707-.707z"
          />
          <path
            d="M8 1a7 7 0 1 0 4.95 11.95l.707.707A8.001 8.001 0 1 1 8 0v1z"
          />
          <path
            d="M7.5 3a.5.5 0 0 1 .5.5v5.21l3.248 1.856a.5.5 0 0 1-.496.868l-3.5-2A.5.5 0 0 1 7 9V3.5a.5.5 0 0 1 .5-.5z"
          />
        </svg>
        Watch Later
      </a>
    </div>
  </div>
  <div
    className="d-flex align-items-center"
  >
    <p
      className="mb-0"
    >
      Made with:
    </p>
    <img
      alt="tmdb"
      className="ms-2"
      src="tmdb.svg"
      width="100px"
    />
  </div>
</header>
`;
